src/lando/urls.py

```diff
@@ -18,9 +18,16 @@
 from django.urls import include, path
 
 from lando.ui.legacy import revisions
+from lando.dockerflow import views as DockerflowViews
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("oidc/", include("mozilla_django_oidc.urls")),
     path("D<int:revision_id>/", revisions.Revision.as_view()),
 ]
+
+urlpatterns += [
+    path("__version__", DockerflowViews.VersionView.as_view(), name="version"),
+    path("__heartbeat__", DockerflowViews.HeartbeatView.as_view(), name="heartbeat"),
+    path("__lbheartbeat__", DockerflowViews.LoadBalancerHeartbeatView.as_view(), name="lbheartbeat"),
+]
```

Comment:
I think it is better to separate them as they are here. In fact it is not uncommon to have each list be named after the app itself and then to add up all the lists at the end, or for some of these URLs to even be defined in a different module and imported/added here. I would suggest that we organize the original list a little better though.

Reply:
Yeah, I'll leave this as-is for now. We can definitely improve the organization even more in the future.

Reply:
In the long run it might be better to define app-specific urls in the app package itself (e.g., `lando.dockerflow.urls`) and then add them here one way or another. Probably a good thing to do in a separate bug though in the future as you said!